include/net/nrf_cloud.h

```diff
@@ -101,21 +101,27 @@ enum nrf_cloud_evt_type {
 	NRF_CLOUD_EVT_PINGRESP,
 	/** The data sent to the cloud was acknowledged. */
 	NRF_CLOUD_EVT_SENSOR_DATA_ACK,
-	/** The transport was disconnected. */
+	/** The transport was disconnected. The status field in the event struct will
+	 *  be populated with a @ref nrf_cloud_disconnect_status value.
+	 */
 	NRF_CLOUD_EVT_TRANSPORT_DISCONNECTED,
 	/** A FOTA update has started. */
 	NRF_CLOUD_EVT_FOTA_START,
 	/** The device should be restarted to apply a firmware upgrade */
 	NRF_CLOUD_EVT_FOTA_DONE,
 	/** An error occurred during the FOTA update. */
 	NRF_CLOUD_EVT_FOTA_ERROR,
+	/** An error occurred when connecting to the nRF Cloud. The status field in the event
+	 *  struct will be populated with a @ref nrf_cloud_connect_result value.
+	 */
+	NRF_CLOUD_EVT_TRANSPORT_CONNECT_ERROR,
 	/** An error occurred. The status field in the event struct will
-	 * be populated with a @ref nrf_cloud_error_status value
+	 *  be populated with a @ref nrf_cloud_error_status value.
 	 */
 	NRF_CLOUD_EVT_ERROR = 0xFF
 };
 
-/** @brief nRF Cloud error status, used to describe NRF_CLOUD_EVT_ERROR */
+/** @brief nRF Cloud error status, used to describe NRF_CLOUD_EVT_ERROR. */
```

Comment:
I'm not 100% sure if linking works here but I'd try and see the generated documentation:
```suggestion
/** @brief nRF Cloud error status, used to describe @ref NRF_CLOUD_EVT_ERROR event. */
```

If this works, same for other enums for event statuses.

Reply:
I'll give it a try.

Reply:
Done and it seems to work.